lib/index.js

```diff
@@ -109,6 +109,7 @@ var ReactTelephoneInput = React.createClass({
     },
     componentDidMount: function componentDidMount() {
         document.addEventListener('keydown', this.handleKeydown);
+        document.addEventListener('focus:' + this.props.name, this.onFocusEventHandler);
```

Comment:
The idea here is to allow external components/events set focus on the nested input field. For example, if you have an 'hidden form' with the react-telephone-input component that is displayed on a button click, you can display the form and set the focus on the nested input field directly.